===from:Rebecca Longmire
===email:rlongmir@iastate.edu
===date:Wed Feb  4 21:46:24 1998
===subject:ACHONDROPLASIA
===message:Achondroplasia is a genetic disorder of bone growth that is evident at birth. It affects about one in every 20,000 births and it occurs in all races and in both sexes. Its depiction in ancient Egyptian art makes it one of the oldest recorded birth defects. <P>
	The word Achondroplasia is derived from Greek and means "without cartilage formation," although individuals with Achondroplasia do have cartilage. During fetal development and childhood, cartilage normally develops into bone, except in a few places, such as the nose and the ears. In individuals with Achondroplasia, something goes wrong during this process, especially in the long bones (such as those of the upper arms and thighs). The rate at which cartilage cells in the growth plates of the long bones turn into bone is slow, leading to short bones and reduced height.<P>
	It is a common  autosomal dominant disorder.  In some cases, a child inherits Achondroplasia from a parent who also has the condition. If one parent has the condition and the other does not, with each pregnancy there is a 50 percent chance that each child will be affected. If both parents have Achondroplasia, there is a 50 percent chance that the child will inherit the condition, a 25 percent chance that the child will not have it, and a 25 percent chance that the child will inherit one abnormal gene from each parent and have severe skeletal abnormalities that lead to early death. A child who does not inherit the gene will be completely free of the condition, and cannot pass it on to his or her own children. <P>
	Achondroplasia is caused by a failure of proliferation and column formation of epiphysial cartilage cells, that is, by a defect in endochondral bone formation which impairs the longitudinal growth of the tubular bones. Point mutations in the FGR3 gene (encoding fibroblast growth receptors) on human chromosome 4 p segregate with disease in achondroplasia families (Rousseau, F., et al. Nature 371:252-254, 1994).<P>
	In most cases (over 80 percent), however, Achondroplasia is not inherited but results from a new mutation (change) that occurred in the egg or sperm cell that formed the embryo. The parents of children with Achondroplasia resulting from new mutations are usually average-sized.  Analysis for the two most common mutation reported in individuals with Achondroplasia (ACH1138G->C and ACH1138 G->A in the FGFR3 gene). (Rousseau, F., et al. Nature 371:252-254, 1994).<P>




Rousseau, F., et al.  Mutations in the gene encoding fibroblast growth factor receptor 3 in 	achondroplasia.  Nature, volume 371. Sept 15, 1994. pages 252 - 254.<P>

National Center for Biotechnology Information. Online Mendelian Inheritance in 	Man.(OMIM) ACHONDROPLASIA. Reference #100800.
	http://www3.ncbi.nlm.nih.gov/htbin-post/Omim/dispmim?100800<P>

=========yorg=========
===from:Andria Bolton
===email:abolton@iastate.edu
===date:Thu Feb 12 13:11:49 1998
===subject:Achondroplasia
===message:	The gene Fibroblast Growth Factor Receptor-3 (FGFR3) is 
found on chromosome 4 (4p16.3).  FGFR3 is known to cause a 
disease called achondroplasia.  Many distortions of the body 
are associated with this common form of short-limb dwarfism.  
Included in these distortions is a long narrow truck, 
shortened extremities, a large head, and a trident hand 
(internet 100800).  It has been seen by researchers that 
risk of death in infants with achondroplasia may approach 
7.5% mainly because of the increased cervical cord 
compression (Hecht et al 1987).<P>
	FGFR3 is inherited as an autosomal dominant.  About 7/8 
are the result of a new mutation (internet 100800).  
Homozygosity of FGFR3 results in severe disorder of the 
skeleton -- early death results (Hall 1969).<P>
	There are two known mutations.  The first location is 
on nucleotide 1138 on codon 380 in the sequence GGG.  Almost 
all of the time the first G changes to an A.  A few cases 
show the first G turning to a C.  The other location is on 
codon 375 where there is a G to T transversion (Shiang 
1994).<P>
<P>
<P>
References cited<P>
<P>
Internet source located at:
http://www3.ncbi.nlm.nih.gov/htbin-
post/Omim/dispmim?100800<P>

Hecht, J.T.; Francomano, C.A.; Horton, W.A.; Annegers, 
J.F.: Mortality in Achondroplasia. Am.J.Hum.Genet. 41: 454-464, 1987.<P>

Hall, J.G.; Dorst, J.P.; Taybi, H.; Scott, C.I., Jr.; Langer, L.O., Jr.; McKusick, V.A.:  Two Probable 
Cases of Homozygosity for the achondroplasia gene.  Birth Defects Orig. Art. Ser. V(4):  24-34, 1969.<P>

Shiang, R.; Thompson, L.M.; Zhu, Y.Z.; Church, D.M.; Fielder, T.J.; Bocian, M.; Winokur, S.T., Wasmuth, J.J.:  Mutations in the transmembrane domain of FGFR3 cause the most common genetic form of dwarfism, achondroplasia.  Cell 78:  335-342, 1994.<P>

=========yorg=========
===psite:http://www.public.iastate.edu/~wahaut
===from:Wes Haut
===email:wahaut@iastate.edu
===institution:Iowa State University
===date:Sat Feb 14 10:34:44 1998
===subject:Neurofibromin
===message:<P>Neurofibromin</P>

<P>Wes Haut</P>

<P>Neurofibrimin is a genetic disorder that is associated with
numerous diseases and defects in the central and peripheral nervous
system (Xu, et al. 1990). Also called NF1, neurofibromin is "is a
common... disorder characterized by abnormalities in multiple tissues
derived from the embryonic neural crest" (Marchuk, et al. 1991). NF1
is located on chromosome 17q11.2, and is autosomal dominant
(Bernards, et al. 1992). Neurofibromin is one of two types of
neurofibromin, the other being neurofibromin 2. Although NF1 has been
called peripheral neurofibromatosis, the disease has also been
associated with disorders of the central nervous system as well. Some
effects include "tumors of the central nervous system, which include
astrocytomas of the visual pathways, ependymomas, meningiomas, and
some primitive neuroectodermal tumors. The most common neuroimaging
abnormality...has been a high signal intensity lesion in the basal
ganglia, thalamus, brain stem, cerebellum, or subcortical white
matter referred to as an 'unidentified bright object' (UBO)...thought
to represent sites of vacuolar change" (Parazzini, et al. 1995).
Neurofibromin has been affiliated with other disorders as well,
including scoliosis, mental retardation and hypertension (Clinical
synopsis, 1998).</P>

<P>Neurofibromin is a dangerous and debilitating disease that is
drawing much attention and research. Due to the diligence and
expertise of numerous scientists, geneticists, and other researchers,
this horrible disease is becoming more understood every day.</P>

<P>Below is a brief listing of the disorders caused by neurofibromin
</P>

<P>(Clinical synopsis, 1998)</P>

<P>Skin :</P>

<BLOCKQUOTE><DIR>
   Neurofibromas
   
   <P>Plexiform neuroma</P>
   
   <P>Cafe-au-lait spots</P>
   
   <P>Axillary freckling</P>
   
   <P>Inguinal freckling</P>
</DIR></BLOCKQUOTE>

<P>Eyes :</P>

<BLOCKQUOTE><DIR>
   Lisch nodules (iris hamartomas)
   
   <P>Glaucoma</P>
   
   <P>Hypertelorism</P>
</DIR></BLOCKQUOTE>

<P>Spine :</P>

<BLOCKQUOTE><DIR>
   Scoliosis
</DIR></BLOCKQUOTE>

<P>Skull :</P>

<BLOCKQUOTE><DIR>
   Sphenoid dysplasia
</DIR></BLOCKQUOTE>

<P>Limbs :</P>

<BLOCKQUOTE><DIR>
   Pseudoarthrosis
   
   <P>Thinning of long bone cortex</P>
   
   <P>Massive leg overgrowth</P>
</DIR></BLOCKQUOTE>

<P>Oncology :</P>

<BLOCKQUOTE><DIR>
   Pheochromocytoma
   
   <P>Meningioma</P>
   
   <P>Optic glioma</P>
   
   <P>Acoustic neuroma</P>
   
   <P>Hypothalamic tumor</P>
   
   <P>Neurofibrosarcoma</P>
   
   <P>Rhabdomyosarcoma</P>
   
   <P>Duodenal carcinoid</P>
   
   <P>Somatostatinoma</P>
   
   <P>Parathyroid adenoma</P>
</DIR></BLOCKQUOTE>

<P>Neuro :</P>

<BLOCKQUOTE><DIR>
   Mental retardation
   
   <P>Aqueductal stenosis and hydrocephalus</P>
</DIR></BLOCKQUOTE>

<P>Endocrine :</P>

<BLOCKQUOTE><DIR>
   Hypertension
   
   <P>Hypophosphatemic osteomalacia</P>
</DIR></BLOCKQUOTE>

<P>Metabolic :</P>

<BLOCKQUOTE><DIR>
   Hypoglycemia
</DIR></BLOCKQUOTE>

<P>Abdomen :</P>

<BLOCKQUOTE><DIR>
   Intraperitoneal mesodermal tumor
</DIR></BLOCKQUOTE>

<P>Vascular :</P>

<BLOCKQUOTE><DIR>
   Arterial occlusive disease
   
   <P>Renal artery stenosis</P>
</DIR></BLOCKQUOTE>

<P>Lung :</P>

<BLOCKQUOTE><DIR>
   Papillary adenomas
   
   <P>Interstitial pulmonary fibrosis</P>
   
   <P>Pulmonary hypertension</P>
</DIR></BLOCKQUOTE>

<P>Inheritance :</P>

<BLOCKQUOTE><DIR>
   Autosomal dominant (17q11.2)
</DIR></BLOCKQUOTE>

<P> </P>

<P>Below is a list of mutations discovered in neurofibromin</P>

<P>	(Allelic variants, 1998)</P>

<P>0001 : Neurofibromatosis, Type I</P>

<P>Mutation : Nf1, Alu Ins</P>

<P>0002 : Neurofibromatosis, Type I</P>

<P>Mutation : Nf1, 5-Bp Del</P>

<P>0003 : Neurofibromatosis, Type I</P>

<P>Mutation : Nf1, Leu?Pro</P>

<P>0004 : Neurofibromatosis, Type I</P>

<P>Mutation : Nf1, Arg365ter</P>

<P>0005 : Neurofibromatosis, Type I</P>

<P>Mutation : Nf1, Lys1423glu</P>

<P>0006 : Neurofibromatosis, Type I</P>

<P>Mutation : Nf1, 1-Bp Ins, Fs1841ter</P>

<P>0007 : Neurofibromatosis, Type I</P>

<P>Mutation : Nf1, 1-Bp Ins, Fs1841ter</P>

<P>0008 : Neurofibromatosis, Type I</P>

<P>Mutation : Nf1, Leu2143met</P>

<P>0009 : Neurofibromatosis, Type I</P>

<P>Mutation : Nf1, Tyr2213asn</P>

<P>0010 : Watson Syndrome</P>

<P>Mutation : Nf1, 42-Bp Dup</P>

<P>0011 : Watson Syndrome</P>

<P>Mutation : Nf1, 80-Kb Del</P>

<P>0012 : Neurofibromatosis, Type I</P>

<P>Mutation : Nf1, Arg1947ter</P>

<P>0013 : Neurofibromatosis, Type I</P>

<P>Mutation : Nf1, Ivs18ds, G-A, +1</P>

<P>0014 : Neurofibromatosis, Type I</P>

<P>Mutation : Nf1, 2-Bp Del</P>

<P>0015 : Neurofibromatosis, Type I</P>

<P>Mutation : Nf1, Met1035arg</P>

<P>0016 : Neurofibromatosis, Type I</P>

<P>Mutation : Nf1, Arg1391ser</P>

<P>0017 : Neurofibromatosis, Type I</P>

<P>Mutation : Nf1, 250-Kb Del</P>

<P><U> </U></P>

<P><U>References</U></P>

<P>Allelic variants. <I>OMIM</I>. Available:
http://www3.ncbi.nlm.nih.gov/htbin-post/Omim/dispmim?162200#VariantList?162200
Accessed: 12 Feb, 1998.</P>

<P>Bernards A, et al. (1992). DNA Cell Biol. Dec;11(10):727-734.</P>

<P>Clinical synopsis. <I>OMIM</I>. Available:
http://www3.ncbi.nlm.nih.gov/htbin-post/Omim/dispmim?162200.cs
Accessed: 12 Feb, 1998.</P>

<P>Marchuk DA, et al. (1991). <I>Genomics.</I> Dec;11(4):931-940.
</P>

<P>Parazzini C, et al. (1995). Spontaneous involution of optic
pathway lesions in neurofibromatosis type 1: serial contrast MR
evaluation. <I>Am. J. Neuroradiol</I>. 16: 1711-1718.</P>

<P>Xu GF, et al. (1990). <I>Cell.</I> Aug 10;62(3):599-608.</P>

<P> </P>

<P> </P>

<P> </P>

<P> </P>

<P> </P>

<P> </P>

<P> </P>
</BODY>
</HTML>


=========yorg=========
===from:Bill Rockey
===email:brockey@iastate.edu
===institution:ISU
===date:Sun Feb 15 14:47:30 1998
===subject:Bloom Syndrome
===message:	A defect in the human gene BLM causes Bloom Syndrome, a congenital disorder characterized by many symptoms, including dwarfism, telangiectasia (chronic dilation of capillaries causing raised dark-red blotches on the skin), and spotty hypo- and hyperpigmentism, to name a few of the more noticeable clinical symptoms.  In addition, sufferers of this condition have highly compromised immune systems, making them highly susceptible to life-threatening infections.  Bloom Syndrome patients also have a learning disability and are prone to cancer.  (OMIM, 1998)  <P>
	The gene BLM is located on chromosome 15, and has an autosomal recessive inheritance pattern.  Bloom Syndrome has three known allelic variations.  The first is a 6 base pair deletion/7 base pair insertion at nucleotide 2281, causing a stop codon a few codons after amino acid 736.  The second allele is a deletion of CAA at nucleotide position 631-633, causing a stop codon at amino acid position 186.  The final variation is a T to C transition at nucleotide 2596 which results in an isoleucine to threonine substitution at amino acid position 843.  (OMIM, 1998) <P>

Online Mendelian Inheritance in Man.  OMIM Entry 210900.  http://www3.ncbi.nlm.nih.gov/htbin-post/
	Omim/dispmim?210900 (15 February 1998).  

=========yorg=========
===from:James Nisly
===email:jnisly@kctc.net
===date:Sun Feb 15 15:25:14 1998
===subject:Polycystic Kidney Disease Type 1
===message:Genetic Diseases Assignment, Part II <P>
by James Nisly <P>

I selected the gene named PKD1 for this assignment.  The name of the disease associated with this gene defect is Polycystic Kidney Disease Type 1 (http://www.nhgri.nih.gov/DIR/GTB/CLONE/; Feb. 15, 1998).  This gene defect in humans produces a systemic disease with cysts in the kidneys and other organs, most commonly in the liver, and occasionally in the pancreas and ovaries and with frequent structural abnormalities in the gastrointestinal tract, the vascular tree, and the cardiac valves.  Complete penetrance of the gene is estimated to occur by 90 years of age.  The pathogenesis of PKD1 has not been established. However, altered epithelial cell growth, secretion, and extracellular matrix have all been shown to occur in PKD1.  
These abnormalities could, in fact, contribute to cyst development and extrarenal manifestations.  Cells are not simply stretched to permit a tubular outpouching to become a cyst; cell numbers must increase.  With electron microscopy, polypoid lesions have been noted throughout the cyst walls in both experimental cystic disease and human PKD1, (Wyngaarden, et al; 1992). <P>

Human symptoms of this disease usually manifest as flank pain or back pain and can be constant or intermittent, mild or severe and may be disabling.  Other symptoms can include complications such as urinary tract infections, renal calculi, or retroperitoneal bleeding.  Early in the course of PKD1, the kidneys can be normal in size with only a few cysts.  Ultimately, the kidneys enlarge and may attain the size of a football, weighing as much as 8 kg.  The end-stage kidney appears to be virtually replaced by cysts throughout the renal parenchyma, (Wyngaarden, et al; 1992). <P>

The gene, PKD1 is located on the human chromosome 16.  It is an autosomal dominant disease and has a worldwide 
prevalence of 1 in 200 to 1 in 1000.  It is the most common hereditary disease in the United States affecting 500,000 people, (Wyngaarden, et al; 1992).  There are 44 known mutations of gene PKD1.  They are located at: 16p13 on the homo sapien #16 chromosome, (http://www.uwcm.ac.uk/uwcm/mg/search/120293.html; Feb. 15,1998). <P>


Bibliography: <P>

The National Human Genome Research Institute:  Inherited Disease Genes Identified by Positional Cloning, 
http://www.nhgri.nih.gov/DIR/GTB/CLONE/; Feb. 15, 1998 <P>

Wyngaarden, James B., Smith, Lloyd H., Bennet, J. Claude.  1992. Cecil Textbook of Medicine. 19th Edition. W. B. 
Saunders Company, Harcourt Brace Jovanovish, Inc., Philadelphia <P>

The Human Gene Mutation Database, Cardiff,  http://www.uwcm.ac.uk/uwcm/mg/search/120293.html; Feb. 15, 1998 <P>



=========yorg=========
===from:Jonathan
===email:amullin@nishna.net
===date:Sun Feb 15 21:42:41 1998
===subject:Cystic Fibrosis
===message: <P>Cystic Fibrosis, the disease formerly known as Cystic Fibrosis of the Pancreas has long been a part of mankind's life. Although this disease was just identified ten years ago 119 alles have been found. The gene responsible for Cystic Fibrosis is known as  CFTR; Cystic Fibrosis Transmembrane Conductance Regulator, located on the seventh chromosome. The recessive inheritance saves many from the ill fated effects of Cystic Fibrosis, but for those who are CFTR homosygotes there is little to look forward. First detection of Cystic fibrosis was very simple, A midwife would lick a babies head to see if it was salty. If salty it was said they die of Pulmonary Congestion. Recently though we have identified CRTR along with many of CFTR’s  symptoms.Problems arise with the exocrin function of the body. The meconium ileus,Billiary cirrhosis, and the sweat glands all affected. Promising studies suggest Ion channels could be a major factor in the symptoms of Cystic Fibrosis. The chloride ion channels in particular seem adversely affected. Other problems such as infertility in both male and female can sometimes arise in Cystic Fibrosis patients. It is unclear exactly how this problem arises but for the male it is assumed that a linked gene may play a factor in the deformation of the Vas Deferens, thus causing infertility. Female infertility likely comes from an excess in cervical mucus.

<P><U>Sources</U>
 <P>McKusick, Victor A. . 6/3/1986. <I>last update 
<P> terry : 1/8/1998 </I>. OMIM ENTRY 219700.
<P>*219700 CYSTIC FIBROSIS; CF
<P>http://www3.ncbi.nlm.nih.gov/htbin-<P>post/Omim/dispmim?219700#AllelicVariant0113?219700. (2/15/98)

<P><U>Addendum:
 <P> ALLELIC VARIANTS</U>

         <P> 0001 : CYSTIC FIBROSIS 
                 <P>   Mutation : CFTR, PHE508DEL
          <P>0002 : CYSTIC FIBROSIS 
                  <P>  Mutation : CFTR, ILE507DEL
        <P>0003 : CYSTIC FIBROSIS 
                   <P> Mutation : CFTR, GLN493TER
         <P> 0004 : CYSTIC FIBROSIS 
                 <P>Mutation : CFTR, ASP110HIS
         <P> 0005 : CYSTIC FIBROSIS 
                  <P>  Mutation : CFTR, ARG117HIS
          <P>0006 : CYSTIC FIBROSIS 
                 <P>   Mutation : CFTR, ARG347PRO
         <P> 0007 : CYSTIC FIBROSIS 
                 <P>   Mutation : CFTR, ALA455GLU
         <P> 0008 : CYSTIC FIBROSIS 
                   <P> Mutation : CFTR, IVS10AS, G-A, -1
       <P>0009 : CYSTIC FIBROSIS 
                   <P> Mutation : CFTR, GLY542TER
        <P>0010 : CYSTIC FIBROSIS 
                 <P>Mutation : CFTR, SER549ASN
         <P>0011 : CYSTIC FIBROSIS 
              <P> Mutation : CFTR, SER549ILE
       <P>0012 : CYSTIC FIBROSIS 
                 <P>   Mutation : CFTR, SER549ARG
      <P>0013 : CYSTIC FIBROSIS 
                 <P>Mutation : CFTR, GLY551ASP
       <P>0014 : CYSTIC FIBROSIS 
                  <P>Mutation : CFTR, ARG553TER
         <P> 0015 : CYSTIC FIBROSIS 
                  <P>Mutation : CFTR, ALA559THR
        <P>  0016 : CYSTIC FIBROSIS 
                <P>Mutation : CFTR, ARG560THR
          <P>0017 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, TYR563ASN
          <P>0018 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, PRO574HIS
          <P>0019 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, AT INS, 2566
          <P>0020 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, C DEL, 3659
          <P>0021 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, SER1255TER
          <P>0022 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, TRP1282TER
          <P>0023 : CFTR POLYMORPHISM 
                    <P>Mutation : CFTR, MET470VAL
          <P>0024 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, ILE506VAL
          <P>0025 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, PHE508CYS
          <P>0026 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, TRP846TER
          <P>0027 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, TYR913CYS
          <P>0028 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, GLY458VAL
          <P>0029 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, TRP1316TER
          <P>0030 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, TC INS, 1154
          <P>0031 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, T DEL, 1213
          <P>0032 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, ASN1303LYS
          <P>0033 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, ARG1162TER
          <P>0034 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, ARG334TRP
          <P>0035 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, 2-BP DEL, 1677TA DEL, FS515TER
          <P>0036 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, ARG851TER
          <P>0037 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, GLY551SER
          <P>0038 : CYSTIC FIBROSIS 
                   <P> Mutation : CFTR, GLY85GLU
          <P>0039 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, ARG1158TER
          <P>0040 : CYSTIC FIBROSIS 
                   <P> Mutation : CFTR, IVS19DS, A-G, +4
          <P>0041 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, 22-BP DEL
          <P>0042 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, 1-BP DEL, 556A DEL, FS
          <P>0043 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, 1-BP DEL, 557T DEL, FS
          <P>0044 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, 84-BP DEL, EX13, 1949DEL84
          <P>0045 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, 1-BP INS, 2869 INS G, FS
          <P>0046 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, VAL520PHE
          <P>0047 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, CYS524TER
          <P>0048 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, GLN1291HIS
          <P>0049 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, PHE311LEU
          <P>0050 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, 2-BP DEL, CT1221
          <P>0051 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, SER492PHE
          <P>0052 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, ARG560LYS
          <P>0053 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, GLU827TER
          <P>0054 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, ARG1066HIS
          <P>0055 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, ALA1067THR
          <P>0056 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, IVS20DS, G-A, +1
          <P>0057 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, 5-BP DUP, 3320 + TCTATG
          <P>0058 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, ARG1066CYS
          <P>0059 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, 1-BP DEL, T1078
          <P>0060 : VAS DEFERENS, CONGENITAL BILATERAL ABSENCE OF 
                    <P>Mutation : CFTR, ASP1270ASN
          <P>0061 : VAS DEFERENS, CONGENITAL BILATERAL ABSENCE OF 
                    <P>Mutation : CFTR, GLY576ALA
          <P>0062 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, IVS19DS, +10 KB, C-T
          <P>0063 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, ARG1283MET
          <P>0064 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, IVS12DS, G-A, +1
          <P>0065 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, GLN359LYS AND THR360LYS
          <P>0066 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, IVS6, 12-BP DEL
          <P>0067 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, ARG347LEU
          <P>0068 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, ALA349VAL
          <P>0069 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, ALA554GLU
          <P>0070 : CYSTIC FIBROSIS 
                   <P> Mutation : CFTR, LYS716TER
         <P> 0071 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, IVS13DS, G-A, +1
          <P>0072 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, GLN1238TER
          <P>0073 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, IVS19AS, G-A, -1
          <P>0074 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, 1-BP INS, 3898 INS C, FS
          <P>0075 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, TRP57TER
          <P>0076 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, GLN1313TER
          <P>0077 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, GLU92LYS
          <P>0078 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, ARG347HIS
          <P>0079 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, GLY91ARG
          <P>0080 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, PHE1286SER
          <P>0081 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, 1-BP INS, 2307 INS A, FS
          <P>0082 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, GLU92TER
          <P>0083 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, GLY480CYS
          <P>0084 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, LEU206TRP
         <P> 0085 : NASAL POLYPS, RECURRENT 
                    <P>Mutation : CFTR, 18-BP DEL, EX4
          <P>0086 : VAS DEFERENS, CONGENITAL BILATERAL ABSENCE OF 
                    <P>Mutation : CFTR, IVS8SA, 5T VARIANT
          <P>0087 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, THR338ILE 
          <P>0088 : CYSTIC FIBROSIS 

                    <P>Mutation : CFTR, TRP1089TER
          <P>0089 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, 4-BP DEL, TATT4010
         <P> 0090 : CYSTIC FIBROSIS 
                   <P> Mutation : CFTR, ILE556VAL
          <P>0091 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, TYR109CYS
          <P>0092 : CYSTIC FIBROSIS 
                   <P> Mutation : CFTR, ARG352GLN
          <P>0093 : CYSTIC FIBROSIS 
                   <P> Mutation : CFTR, IVS3, DS, A-G, +4
          <P>0094 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, GLN524HIS
          <P>0095 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, GLU542TER
          <P>0096 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, GLN552TER
          <P>0097 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, ASP648VAL
          <P>0098 : CYSTIC FIBROSIS 
                   <P> Mutation : CFTR, LYS710TER
          <P>0099 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, GLN890TER
          <P>0100 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, SER912LEU
          <P>0101 : CYSTIC FIBROSIS 
                   <P> Mutation : CFTR, 2-BP DEL, TA936
          <P>0102 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, HIS949TYR
          <P>0103 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, LEU1065PRO
          <P>0104 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, GLN1071PRO
          <P>0105 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, HIS1085ARG
          <P>0106 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, TYR1092TER
          <P>0107 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, TRP1204TER
          <P>0108 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, 1-BP DEL, G1215
          <P>0109 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, THR1220ILE
          <P>0110 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, ILE1234VAL
          <P>0111 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, GLY1249GLU
          <P>0112 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, SER1251ASN
          <P>0113 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, SER1255PRO
          <P>0114 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, ASN1303HIS
           <P>0115 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, 2-BP DEL, CA1609
          <P>0116 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, 1-BP DEL, G2423
          <P>0117 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, 1-BP DEL, A3293
          <P>0118 : CYSTIC FIBROSIS 
                    <P>Mutation : CFTR, 4-BP INS, TCAA3667




=========yorg=========
===from:Dawn Spiller
===date:Sun Feb 15 21:44:44 1998
===subject:Breast Cancer-type 1
===message:	Breast Cancer affects thousands of women.  It can have devastating effects of death or mastectomy.  Breast cancer is a adenocarcinoma (Encarta).  Adenocarcinomas are the cancerous tumors which can develop on glands of the mammary and prostate.  Malignant cancer cells develop a tumor on the mammary gland, and destroy the tissue around the gland.  Since I know a lot of women with this terrible disease, I decided to do the gene BRCA1 which causes breast cancer type 1 or early onset breast-ovarian cancer.<p>
	The gene BRCA1 is located on chromosome 17.  It is an autosomal(non-sex determining) dominant allele.  There are 21 mutations of this gene in various nucleotide positions.  One substitutes a glycerine for a tyrosine.  The gene encodes a protein of 1,863 amino acids and is a tumor suppressor gene.  A mutation would increase the risk of abnormal growth.  Breast cancer type 1 is early onset meaning women develop the disease at a young age (20-43). 925 of the women having the dominant allele develop the cancer. Male carriers have and increased risk of prostate cancer (NHGRI).<p>
=========yorg=========
===from:Christina Naumann-Mrizek
===date:Sun Feb 15 23:11:43 1998
===subject:McLeod Syndrome
===message:McLeod Syndrome<P>McLeod Syndrome is an X-linked multisystem disorder characterized by abnormalities in the neuromuscular and hematopoietic systems (Ho, et al., 1996). In this disorder, red blood cells demonstrate acanthocytic morphology and weakened antigenicity in the Kell blood group system (Bertelson et. al., 1988). McLeod syndrome also reduces the red cell survival in the body, some compensation may be made by increased hyperplasia of red blood cells.<P> McLeod Syndrome is caused because of the absence of the Kx antigen which is responsible for normal cell morphology (Wimer, 1977). The faulty allele is found on the Xk gene on the X chromosome. The disorder appears because of a point mutation in exon 2 that creates a frame shift and results in premature termination of the translation process and formation of a faulty Kx antigen (Ho et al, 1996).<P> References:<P>Bertelson CJ, Pogo AO, Chaudhuri A., Marsh WL, Redman CM, Banerjee D, Sy
=========yorg=========
===from:Lori Applegate
===email:lorapp@iastate.edu
===institution:ISU
===date:Mon Feb 16  8:31:50 1998
===subject:Adrenoleukodystrophy
===message:Adrenoleukodystrophy<P>

Adrenoleukodystrophy (ALD) is a serious disease which affects approximately 1 in 100,000 people with most of them male.  This disease is X-linked recessive and is believed to be the ALD gene.  It’s located on the locus of the X chromosome--Xq28  (“Adrenoleukodystrophy”).<P>

There are two forms of the ALD disease:  (1) a childhood form that leads to total disability or death the first ten years of life and (2) an adult form (also called Adrenomyeloneuropathy) that progresses slower and allows the individual to live a near-normal life.  In the childhood form, early symptoms of neurological damage include hyperactivity, difficulty with speech, impaired spatial orientation, visual problems, and seizures.  Later, symptoms persist with a complete loss of speech, inability to swallow, increased paralysis, and endocrinological problems which leads to a vegetative state. Neurological damage occurs in the adult form which includes degeneration of the spinal cord, and peripheral nervous system, progressive loss of cognitive ability and endocrinological symptoms (DeLaCruz, Kho, and Tran).<P>

There have been various mutations hypothesized for Adrenoleukodystrophy.  The main one is a substitution of an Adenine base for a Guanine base in codon 617 of the ALD gene.  A mis-sense point mutation (mutation resulting in the coding of the wrong amino acid) on exon 6 and a non-sense mutation (a mutation that encodes a stop codon) on exon 4.  Another is a 38 base pair deletion causing extensive mutations and a 3 base pair deletion that results in the loss of the use of ATP binding domain of an ALD protein (DeLaCruz, Kho, and Tran).<P>

Unfortunately there is no cure for this disease.  There are a few treatments, but are not very effective and cannot reverse damage already done to the individual.  Bone marrow transplants have been tried, but have had poor results  (DeLaCruz, Kho, and Tran).<P>

Bibliography<P>

“Adrenoleukodystrophy.”  Pediatric Database (PEDBASE) Homepage.  25 July 1994.
http://www.icondata.com/health/pedbase/files/ADRENOLE.HTM  (15 February 1998).<P>

DeLaCruz, Rene;  Kho, David;  Tran, Chuyen.  “Adrenoleukodystrophy (ALD):  A Peroxisomal Disorder.”  University of Washington.  2 April 1996.  
http://www.biology.washington.edu/bsa/bio...Papers/AB%20Adrenoleukodystrophy%202.html  (12 February 1998).<P>

=========yorg=========
===from:David Cohen
===email:cato@iastate.edu
===date:Mon Feb 16 11:44:34 1998
===subject:Huntington's Disease
===message:Huntington's Disease (HD) is a degenerative brain disease begining at mid-life.  "HD is caused by the expansion of polymorphic trinucleotide repeat (CAG)n located in the coding region of the gene for huntingtin.  HD occurs when the cells in the caudate nucleus of the brain begin to die.  HD causes relentless deteriation of intelectual ability, emotional control, balance and speech.  Chorea or involuntary movements is nearly always a symptom as well.  

HD is caused by a single dominant gene from either parent.  A child with one parent that suffers from HD has a 50% chance of obtaining that gene.  

The HD gene was located in 1983, but not correctly identified until 1993.  Its location was pinpointed to chromosome number four.  There are fifty known mutations in the HD gene.

Bibliography:

Huntington's Disease.  Available at: www.interlog.com/~rlaycock/2nd.html

National center for Biotechnology.  Available at:  www3.nlm.nib.gov:80/htbin-post/omim/dispmim?143100  
=========yorg=========
===from:Sheri Staudt
===email:sstaudt@iastate.edu
===date:Mon Feb 16 12:14:44 1998
===subject:Genetic Disorder
===message:		Achondroplasia, also called chondrodystrophia fetalis, is 
a form of dwarfism.  This particular disorder is, unlike most 
harmful alleles, dominant instead of recessive.  There is one 
case of this dominant allele disorder in every 10,000 people 
(Campbell, 1996).  The people who have the phenotype for
dwarfism are heterozygous.  The individuals who are homozygous
for this recessive allele and not achondroplastic dwarfs make 
up 99.99% of the human population (Campbell, 1996).  Since
these dominant alleles cannot be masked in heterozygous pairs,
they are much less common than recessive pairs.  Most of these
dominant alleles stem from mutations in a sperm or egg gene.
Because the mutation is so different from the original gene,
the developing offspring can sometimes spontaneously abort.

			Dwarfism is present when the cartilage cells of bones grow
incorrectly.  Characteristics of dwarfs include underdeveloped
arms, legs, and torso.  Those with proportionate size of body
parts are also called midgets (World Book, 1995).  Achondroplastics
are lacking in cartilage cells.  When not enough cartilage is
present, the body's bones are unable to grow correctly.  This
is the most common form of dwarfism, and the torso of the 
individual is of almost normal size.  They also have larger heads
due to overgrowth of bones after the sutures at the skull's base
prematurely fuse.  These individuals may also have bulging
foreheads, protruding jaws, narrow chests, and a deep curve in
the lower back.  Females with this disorder have much difficulty
in childbirth from the narrowness of their hips.  Fortunately, though,people with
this affliction have normal intelligence and otherwise good
health (Britannica, 1995).

Sources:

Campbell, Neil A.  Biology.  California: The Benjamin/Cummings
     Publishing Company, Inc., 1996.

New Encyclopedia Britannica, The.  Achondroplasia, vol. 1, pp. 63.
     Chicago: Encyclopedia Britannica, Inc., 1995.

World Book Encyclopedia, The.  Dwarf, vol. 5, pp. 394.  Chicago:
     World Book, Inc., 1995.
=========yorg=========
===from:Roxanne Sweney
===email:rox@iastate.edu
===date:Mon Feb 16 14:46:43 1998
===subject:Obesity: a Chronic Illness
===message:<P>
	The genetic disease I selected was the disease of Obesity, caused by the gene named the (OBS) or the "OB" gene as was described in Friedman, 1991.  Obesity is a chronic disease. A chronic disease is an illness that will remain with a person for life. Looking thin is not a cure but just a result of controlling the chronic illness of obesity. Therefore, even if one loses weight and doesn't "look obese", they are still considered obese.  Obesity is not curable - at least not yet, but sometimes obesity can be successfully controlled.  The metabolic balance point for fat is thought to be set by the brain just like normal temperature, blood pressure, blood sugar level, thyroid level, etc.  The metabolic rate of fat essentially regulates the amount of fat stored and used by the body. There are many factors that affect the metabolic rate of fat. Genetics are the major factor affecting this rate. Other factors seen in various degrees are: Age,  gender, pregnancy, starvation, exercise and medications (Iversen, 1997)
<P>
	According to Friedman (1994), a study of somatic hybrid cell DNAs indicates that the human OB gene is on chromosome 7.  The disease is inherited, as Montague et al. states,  in that "autosomal recessive major locus and polygenic loci account for about 75% of variance " (1997).
In an inbred Pakistani kindred, Montague et al. (1997) found that 2 " morbidly obese" cousins, a female (8) and a male aged 2 years, had a deletion of a guanine nucleotide in the leptin gene. There were 5 rather than the expected 6 guanine nucleotides present between nucleotides 393 and 398.  Therefore, the  mutation  disrupted the reading frame of the leptin gene, leading to the introduction of 14 abnormal amino acids after gly132 in the leptin polypeptide, followed by a premature stop codon. The allelic variance is therefore accounted for as "0.0001 obesity, severe, due to leptin deficiency [LEP, 1-BP DEL, FS147TER ]"
<P>
References:
<P>
 Iversen, Michael,  M.D. (1998).  Obesity: A Contemporary Look at The Proper Diagnosis. 	Obesity. http://www.flash.net/~familymd/obesity.htm 
<P>
 Friedman, J. M.; Leibel, R. L.; Siegel, D. S.; Walsh, J.; Bahary, N., 1991. 
     	Molecular mapping of the mouse ob mutation. Genomics         11: 1054-1062. 
     	
<P>
	Montague, C. T.; Farooqi, I. S.; Whitehead, J. P.; Soos, M. A.; Rau, H.; Wareham, N. J.; Sewter, C. P.; Digby, J. E.; Mohammed, S. N.; Hurst, J. A.; Cheetham, C. H.; Earley, A. R.; Barnett, A. H.; Prins, J. B.; O'Rahilly, S., 1997.  Congenital leptin deficiency is associated with severe early-onset obesity in humans. Nature 387: 903-908. 

=========yorg=========
===from:Matt Sogard
===date:Mon Feb 16 14:54:11 1998
===subject:Situs Inversus
===message:	HTX1, Heterotaxy-1, is a gene located on Homo Sapien chromosome number 14.  The disease known as situs inversus is caused by a mutation of this gene. (OMIM Home Page, 1998)  The word “situs,” refers to the position of the atria.  Situs inversus is the left to right reversal of the cardiac chambers.  The right atrium and ventricle are located on the left side of the heart while the left atrium and ventricle are located on the right of the heart.  This cardiac inversion can also be accompanied by an inversion of the other internal organs such as the liver, spleen, and stomach. (Situs Inversus, 1998) <P>
	One of two known mutations of the HTX1 gene can cause situs inversus.  One type of mutation is a missense mutation in the HTX1 gene caused by a 1502C-T transition.  The other type is a nonsense mutation caused by a 1408C-T transversion. (OMIM Home Page, 1998) <P> 
	Situs inversus is rare, but people born with situs inversus have increased chances of developing congenital heart disease.  HTX1 is an 
x-linked gene.  It is autosomally recessive in its inheritance patterns. (The Genome Database, 1998)<P>

References<P>
The National Human Genome Research Institute.
 www.nhgri.nih.gov/DIR/GTB/CLONE/
 February 13, 1998<P>
National Center for Biotechnology Information, OMIM Home Page
 www3.ncbi.nlm.nih.gov/Omim/
 February 13, 1998<P>
The Genome Database
 gdbwww.gdb.org/
 February 13, 1998<P>
Situs Inversus
 www.ohsu.edu/cliniweb/C16/C16.131.810.html
 February 13, 1998<P>

=========yorg=========
===from:Dallas 
===email:kcrowley@nwiowa.com
===institution:AC-T SD
===date:Mon Feb 16 15:36:41 1998
===subject:Chediak-Higashi
===message:Chediak-Higashi syndrome, or CHS, includes symptoms such as partial albinism, silvery-
blond hair, photophobia, and peculiar malignant lymphoma.  There is also an
immunodeficiency that causes patients to get recurrent infections, and the patient is usually
dead by the age of seven ("Clinical synopsis. . ." 98).  This syndrome is due to a problem
with the lysosomal traffiking regulator gene, or LYST.  The location of CHS has been
mapped to the first chromosome at segment q42.1-q42.2 with autosomal recessive
inheritance, and seven allelic varients have been discovered.  Six childhood types are as
follows:  1-BP DEL;  1-BP INS;  1-BP INS, FS638TER;  1-BP DEL, FS3258TER; ARG50TER; and GNN1029TER.  There is also one late-onset type:  ARG1103TER (Barrat et al 96).  There are ten species known to exhibit this disease.  They include mouse, mink, cat, cattle, killer whale, man, and four other species (Kahraman and Prieur 90).<P>

Bibliography:<P>

Barret, F. J.; Auloge, L.; Pastural, E.; Dufource Lagelouse, R.; Vilmer, E.; Cant, A. J.; Weissenbach, J.; LePaslier, D.; Fischer, A.; de Saint Basille, G.:  "Genetic and physical mapping of the Chediak-Higashi syndrome on chromosome 1q42-43." American Journal of Human Genetics.  59:  625-632, 1996.<P>

"Clinical Synopsis for OMIM ENTRY 214500."  Online. Internet.  2/16/98
http://www3.ncibi.nlm.nih.gov/htbin-post/Omim/dispmim?214500.cs<P>

Kahraman, M. M.; Prieur, D. J.:  "Chediak-Higashi syndrome in the cat:  prenatal diagnosis by amniote fluid in cells."  American Journal of Medical Genetics.  36:  321-327, 1990.<P>
=========yorg=========
===from:Eric Underbakke
===email:eundrbak@iastate.edu
===date:Mon Feb 16 16:28:10 1998
===subject:Huntington's Disease
===message:Huntington's Disease<P>
Eric Underbakke<P>

	Huntington's disease is a rare and fatal genetic disease first diagnosed by George Huntington in 1872 in his paper called "On Chorea"(Jankovic, 1997).  Nearly 25,000 people are currently suffering the effects of Huntington's disease, and 150,000 have the mutant gene(Henahan, 1995)<P>
	The gene causing Huntington's disease is located on the 4th chromosome on the end of the short arm.  The gene itself is called ‘huntingtin'(OMIM, 1998).  The affliction is an autosomal dominant trait, so every person carrying the mutant gene will eventually show symptoms of the disease, if they live long enough.  The actual mutation is an increase in the number of CAG repeats in the huntingtin gene.  Normal people have between 9 to 37 repeats, while people showing symptoms of Huntington's disease usually have between 37 and 86 CAG repeats(Shapira, 1997).  The exact mechanism that causes the disease is unclear, however, the disease is known to be a progressive loss of mental functioning resulting from brain cell deaths in the caudate nucleus, the most medial of the four basal ganglia in the brain.  The brain cell death is accompanied by a loss of some neurotransmitters and an increase in the levels of dopamine in the brain(OMIM, 1998).<P>
	The symptoms of Huntington's disease usually appear between the ages of 30 and 40.  The affliction is characterized by choleric movements (rapid, involuntary, movement flowing from one body part to another), dementia (personality changes, apathy, paranoia, hostility, hallucinations), rigidity, and sometimes seizures(Jankovic, 1997).  Victims of Huntington's disease are usually institutionalized and die within 10 to 20 years from aspiration, infection, and poor nutrition.<P>
	Mutations of the huntingtin gene can occur when the gene is passed to a sperm cell.  The number of CAG repeats generally increases as the gene is passed on to germ cells(OMIM, 1998).  Scientists have found that more CAG repeats in the gene, the earlier the symptoms appear, and earlier the victim dies.  Interestingly, when the gene is passed to an egg (ie the disease comes from the mother), the mutation is far less extensive.  The CAG repeats slightly increase in number, but the effects on an afflicted offspring are negligible.  The offspring will still have Huntington's disease, but the disease will not appear any earlier than it did in the mother(Shapira, 1997).<P>

Literature Cited:<P>
		Henahan, Sean.  (1995).  Huntington's Disease Protein Found. http://www.gene.com/ae/WN/SU/hunting.html.  (15 Feb.  1998)<P>

		Jankovic, Joseph, M.D. (1997).  Clinical Presentation of Huntington's Disease. http://www.bem.tmc.edu/neurol/struct/hunting/huntp2.html.  (15 Feb.  1998)<P>

		National Center for Biotechnology Information. (1998).  Online Mendelian Inheritance in<P> Man (OMIM).  143100 Huntington Disease; HD.  http://www3.ncbi.nlm.nih.gov/htbin-past/omim/dispmim?143100#Reference189.
		(12 Feb. 1998).<P>


		Shapira, Stuart K.  M.D., Ph.D.  (1997).  Clinical Genetics of Huntington's Disease. http://www.bcm.tmc.edu/neurol/struct/hunting/huntp3.html.  (15 Feb.  1998).<P>
=========yorg=========
===from:Martina Rusch
===email:marti@iastate.edu
===date:Mon Feb 16 16:52:37 1998
===subject:HNPCC
===message:Hereditary non-polyposis colon cancer(HNPCC) is a common hereditary disease characterized by a predisposition to the early onset of colorectal cancer.  The majority of HNPCC families carry germline mutations of either hMSH2 or hMLH1 genes, where as germline mutations of hPMS1 and hPMS2 genes have rarely been observed.(Wang 1997)  HNPCC has been found to be a common autosomal dominant disease caused by these germline mutations in the DNA mismatch repair genes.(Luce 1996)  This mutation in the DNA mismatch repair gene hMLH1 is linked to the chromosome 3p.(Bronner 1994)<P>

The hereditary cancer diagnosis is often missed because the family history of cancer is given short shrift in medical practice.  Hereditary nonpolyposis colorectal cancer(HNPCC) certainly fits this estimate, although some studies suggest that a minimum of 2% with a range as high as 10% of the total colorectal cancer burden is due to HNPCC.(Lynch 1997)<P>

Wang, Q.  "Germline hMSH2 and hMLH1 gene mutations in incomplete HNPCC families." Int J Cancer.  1997.
http://www.ncbi.nlm.nih.gov/htbin-post/Entrez/query?uid=9399661&form=6&db=m&Dopt=b<P>

Luce, MC.  "Identification of DNA mismatch repair gene mutations in hereditary nonpolyposis colon cancer patients."  Int J Cancer.  1996.
http://www.ncbi.nlm.gov/htbin-post/Entrez/query?uid=8600061&form=6&db=m&Dopt=b<P>

Lynch, HT.  "Etiology, natural history, management and molecular genetics of hereditary nonpolyposis colorectal cancer (Lynch syndromes): genetic counseling implications."  Cancer Epidemiol Biomarkers Prev.  1997.
http://www.ncbi.nlm.gov/htbin-post/Entrez/query?uid=9499801&form=6&db=m&Dopt=b<P>

Bronner, C.E.  "Mutation in the DNA mismatch repair gene homologue hMLH1 is associated with hereditary nonpolyposis colon cancer linked to chromosome 3p."  Nature.  1994.
http://www.ncbi.nlm.nih.gov/htbin-post/Entrez/query?db=n&form=6&dopt=g&uid=u07343<P>

=========yorg=========
===from:Nikkoli Dollinger
===date:Mon Feb 16 17:27:20 1998
===subject:Huntington Disease
===message:	Huntington Disease is inherited as an autosomal dominant disease(http://www3.ncbi.nlm.nih.gov/htbin-post/omin/dispmin?143100).  Huntington Disease iscaused by mutation in a gene called “huntington”, this gene is located on chromosome4p16.3 (http://www3.ncbi.nlm.nih.gov/htbin-post/omin/dispmin?143100 ).<p>  	The huntington gene contains a section where the same three molecules repeatseveral times (http://ww2.med.jhu.edu/bhdc/hdinfo.html).  In a normal person the genecontains between 11 and 32 copies of this triple repeat, the mutation that causesHuntington Disease increases the repeat to 37 times or more (Myers). <p>	Huntington Disease is a neurodegenerative disease that causes progressive damageto the nervous system (http://ww2.med.jhu.edu/bhdc/hdinfo.html).  The average age ofonset for Huntington’s Disease can vary from 4 to more than 70 years of age(Myers).  Theclassic signs of Huntington Disease are a movement disorder, cognitive change, and achange in personality (Myers)<p>About Huntington’s disease.http://ww2.med.jhu.edu/bhdc/hdinfo.html (15 Jan 98).<p>Myers, R.H. Predictive HD Testinghttp://www.circ.uab.edu/sergg/s95/s95-3.htm (15 Jan 98).<p>OMIN ENTRY 143100.*143100 Huntington Disease; HDhttp://www3.ncbi.nlm.nih.gov/htbin-post/omin/dispmin?143100 (15 Jan 98).<p>The National Human Genome Research instituteInherited Disease Genes Identified by Positional Cloninghttp://www.nhgri.gov/DIR/GTB/Clone/ (15 Jan 98).<p>
=========yorg=========
===from:Ellie Nunez
===date:Mon Feb 16 20:03:52 1998
===subject:Genetic Disease Assignment-Part II
===message:      Juvenile glaucoma is an eye disorder. It is an uncommon form of open angle glaucoma, that usually sets in during childhood or early adulthood and often shows a strong family history (OMIM, 1997).  The most striking features of victims affected with this illness have dark-colored irises; The iris lacks the usual stromal pattern and has a smooth apprearance (OMIM, 1997). Symptoms include myopia, optic nerve deterioration and elevations in intraocular pressures (OMIM,1997).  Some cases are severe enough that if the patient does not get the problem surgically corrected, they risk total eyesight loss (OMIM, 1997). It is of recessive inheritance, located on the cytochrome p450 CYP1B1 gene (OMIM, 1997).



References: 
Online Mendelian Inheritance in Man. February 16, 1997.   
=========yorg=========
===from:Rohit Ramaswamy
===email:rohitr@worldnet.att.net
===institution:Service Design Solutions
===date:Mon Feb 16 21:46:27 1998
===subject:Genetic Disease Assignment - Part II - juvenile glaucoma GLC1A
===message:HEREDITARY JUVENILE OPEN ANGLE GLAUCOMA (GLC1A type) <P>

Glaucoma is a major cause of blindness and is characterized by progressive degeneration of the optic nerve and is usually associated with elevated intraocular pressure. Though glaucoma is typically a disease of late adulthood, an uncommon hereditary form manifests itself in childhood or early adulthood. This form, called hereditary juvenile open angle glaucoma type GLC1A (OMIM ID#137750), has a dominant, autosomal pattern of transmission. This disease should not be confused with another form of juvenile glaucoma (OMIM ID# 231500) that is inherited recessively. It should not also be confused with primary infantile glaucoma (type GLC3A, OMIM ID# 231300 - see the assignment submitted by Ellie Nunez for the gene that causes this disease) or with adult-onset glaucoma, which occurs in the late forties or the early fifties (OMIM ID# 137760). <P>

In a variety of studies conducted over a period of 50 years (see, for example, Zorab(1932), Martin and Zorab(1974), Johnson et al. (1993), Lotufo et al. (1989), Morissette et al. (1995)), researchers found that those affected by the disease exhibited a variety of symptoms. In some cases, affected persons display abnormal pigmentation of the irises, with the irises appearing unusually dark, though it is unclear whether these are actually caused by glaucoma, or by a complementary disorder called IGDA (iridogoniodysgenesis, OMIM ID# 601631). Many of those affected are myopic, and the disease appears to be more prevalent among Americans of African ancestry. However, the most common symptom is high intraocular pressure, that needs to be to be reduced surgically to prevent loss of vision. <P>

In the past few years, researchers such as Johnson et al.(1993), Sheffield et al.(1993) and Wiggs et al. (1994) discovered from several independent samples that the gene for juvenile glaucoma maps to the location 1q21-q31 on the first human chromosome. This gene is referred to in the literature as GLC1A. More recently, Escribano et al. (1995) and Stone et al. (1997) discovered that a gene that was isolated from the ocular ciliary body (a tissue responsible for regulating aqueous humor secretion and intra-ocular pressure) mapped to the same 1q region as the glaucoma gene GLC1A. This gene encodes a protein called Myocilin and therefore is referred to in the literature as MYOC (or TIGR, after "trabecular meshwork-induced glucocorticoid response protein").  Moreover, Stone et al. (1997) found that one of three mutations in the MYOC gene was significantly greater in a sample of glaucoma patients than in a corresponding control sample (3.9% to 0.2%). Stone hypothesized that the mutations in the MYOC gene may increase intra-ocular pressure by obstructing aqueous outflow from the eye, or by causing some muscle abnormality in the ciliary body. Subsequently, Adam et al. (1997) described 5 new mutations in the MYOC gene in 8 French families with juvenile glaucoma. The evidence seems to suggest, therefore, that the MYOC and the GLC1A genes are identical, and that mutations in this  gene could be a possible cause for juvenile glaucoma. In the Genome Data Base (GDB), the records for the MYOC and the GLC1A genes have been merged.<P>

The protein coded by the MYOC gene is a 497-amino acid polypeptide. As mentioned above, eight allelic variants have been discovered. They are:<P>

1) tyr430-to-his <P>
2) gly357-to-val <P>
3) gln361-to-ter <P>
4) pro370-to-leu <P>
5) ile477-to-ser <P>
6) asn480-to-lys <P>
7) pro370-to-leu <P>
8) gly367-to-arg <P>

Note: even though 8 alleles are shown in the database, 4 and 7 appear to be identical - so only 7 seem to be present. <P>

In summary, significant progress has been made in identifying a gene whose mutations appear to be responsible for causing juvenile glaucoma. According to Stone, it is estimated that nearly 100,000 individuals in the U.S. may be suffering from this disease. This would make GLC1A-cause glaucoma the most molecularly recognizable form of blindness. <P>

References <P>

Adam, M. F.; Belmouden, A.; Binisti, P.; Brezin, A. P.; Valtot, F.; Bechetoille, A.; Dascotte,
J.-C.; Copin, B.; Gomez, L.; Chaventre, A.; Bach, J.-F.; Garchon, H.-J. , "Recurrent mutations in a single exon encoding the evolutionarily conserved olfactomedin-homology domain of TIGR in familial open-angle glaucoma.",  Hum. Molec. Genet. 6: 2091-2097, 1997. <P>

Escribano, J.; Ortego, J.; Coca-Prados, M., "Isolation and characterization of cell-specific cDNA clones from a subtractive library of the ocular ciliary body of a single normal human donor: transcription and synthesis of plasma proteins",  J. Biochem. 118: 921-931, 1995. <P>

Johnson, A. T.; Drack, A. V.; Kwitek, A. E.; Cannon, R. L.; Stone, E. M.; Alward, W. L. M.,      "Clinical features and linkage analysis of a family with autosomal dominant juvenile
 glaucoma",  Ophthalmology 100: 524-529, 1993. <P>

Lotufo, D.; Ritch, R.; Szmyd, L., Jr.; Burris, J. E., "Juvenile glaucoma, race, and refraction", J.A.M.A. 261: 249-252, 1989. <P>

Martin, J. P.; Zorab, E. C. , "Familial glaucoma in nine generations of a South Hampshire family", Brit. J. Ophthalmology 58: 536-542, 1974. <P>

Morissette, J.; Cote, G.; Anctil, J.-L.; Plante, M.; Amyot, M.; Heon, E.; Trope, G. E.;
Weissenbach, J.; Raymond, V. "A common gene for juvenile and adult-onset primary open-angle glaucomas confined on chromosome 1q",  Am. J. Hum. Genet. 56: 1431-1442, 1995. <P>
     
Stone, E. M.; Fingert, J. H.; Alward, W. L. M.; Nguyen, T. D.; Polansky, J. R.; Sunden, S. L.
F.; Nishimura, D.; Clark, A. F.; Nystuen, A.; Nichols, B. E.; Mackey, D. A.; Ritch, R.; Kalenak, J.
W.; Craven, E. R.; Sheffield, V. C., "Identification of a gene that causes primary open angle glaucoma",  Science 275: 668-670, 1997. <P>

Zorab, A.,  "Glaucoma simplex familialis",  Trans. Ophthal. Soc. U.K. 52: 446-460, 1932. <P>


=========yorg=========
===from:jay berkey
===date:Mon Feb 16 22:23:18 1998
===subject:genetic diseases assignment - part 2
===message:UBE3A- Ubiquitin-Protein Ligase<P>

	Ubiquitin-Protein Ligase, UBE3A, is the gene at which Angelman Syndrome, AS, arises.  The gene is located on the 15th human chromosome (Kishino, T., M. Lacande, and J. Wagstaff, 1997).<P>  Angelman Syndrome results from a lack of maternal contribution of chromosome 15q11-q13 (Online Mendelian Inheritance in Man, 1997).  This occurs from de novo, repeated, deletion. A mode of inheritance in rare cases, families having more than one occurrence, is autosomal dominant modified by imprinting (Online Mendelian Inheritance in Man, 1997).<P>  
	AS is characterized by mental and motor retardation, seizures, paroxysmal (fitful) laughter, absence of speech, and ataxia (lack of coordination).  The components of ataxia and paroxysmal laughter are what led to early diagnoses of “happy puppet” syndrome (Online Mendelian Inheritance in Man, 1997).  Only in the past twenty years has sensitivity to sufferers and their families been accorded by the name Angelman Syndrome.<P>




Works Cited <P>

Online Mendelian Inheritance in Man.  Angelman Syndrome.  National 	Center for Biotechnology Information.  http://www3.ncbi.nlm.
	nih.gov/htbin-post/Omim/dispmim?105830#TEXT, 2/16/98.<P>

Kishino, T., M. Lalande, and M. Wagstaff.  “UBE3A/E6-AP Mutations 		Cause Angelman Syndrome.”  Nat Genet.  January, 1997; 			15(1):70-73.      	 				     
	
=========yorg=========
===from:jeremy duehr
===date:Mon Feb 16 23:05:44 1998
===subject:part II
===message:Choroidermia Gene.  


Blindness is a not a really serious problem in today's society however it can still be very challenging. Geneticists are constantly searching for cures to blindness and by isolating genes which cause this debilitating disease they come closer to that goal. The X-linked gene Choroidermia(Chm) is one such gene that causes disease of the retina. The blindness it causes is characterized by progressive dystrophy of the Chroid, Retinal pigment epithelium, and Retina.(Van Bokhoven) Choroidermia is located on chromosome X and the mutations, which cause it to be debilitating, are 4 Exons of the 3' portion.(Van Bokhoven)



Van Bokhoven, H
Direct Submission
March 11, 1994
Netherlands

=========yorg=========
===from:Seth Grote
===email:setgrote@iastate.edu
===date:Mon Feb 16 23:35:11 1998
===subject:Menkes Syndrome
===message:Menkes Syndrome #309400 <p>
<br>
Menkes Syndrome is a mutation in the gene encoding Cu(2+), transporting ATPase, alpha polypeptide. <p>
<br>
This disease was found in a family of English-Irish living in New York.  It is discribed as an X-linked recessive disorder characterized by early retardation in growth, peculiar hair, and focal cerebral and cerebellar degeneration. <p>
<br>
The gene order is: Xpter-MS-L1.28-MGU22. <p>
<br>
http://www3.ncbi.nlm.nih.gov/
=========yorg=========
===from:Damon Johnson
===date:Mon Feb 16 23:56:46 1998
===subject:Hey Mr. Wilson
===message:<p>Most of us remeber Mr. Wilson as the Grumpy old man on dennis the Menace, but Wilson Disease is a fatal disease is fatal if not treatred immediately. Wilson' disease is an autosomal recessive disease and its gene is called ATP 7B (ManOMIM).  Wilson's diseases causes the body to accumuklate copper in the brain liver and other vital organs (Taylor).  
<P>The gene ATP 7B creates several different forms of Wilson's disease.  They have identified 16 novel mutations, 7 missense mutations, and 1 splicing defect (ManOMIM).  The disease also has 8 different alleles which range from .0001 to .0008(ManOMIM).  The gene can be found on chromosome number 13 in the 14.3 region(ManOMIM)
<P> If caught soon enough, victims of Wilsons Disease can be cured.  The treatment consits mainly of the use of the drug penicilleamine(Taylor).  If not treated most victims won't live to be 30 years old.
<P>Literature Citied
<P>ManOMIM 141300 Wilson's Disease WD http://www.3nchi.nih.gov/htbin-posst/Omim/dispmim?277900
<P>Taylor, Irene C. http://www.dysphasia.com
=========yorg=========
===from:Rebecca Bollin
===date:Tue Feb 17  0:37:57 1998
===subject:Genetic Diseases Assignment PART II
===message:Hemochromatosis

Hemochromatosis is a condition in which the body absorbs an excess amount of iron, causing an iron overload (Bacon, 96). Iron accumulates in the body over time. This excess iron is stored in various organs, mainly in the liver, but the pancreas, endocrine glands and joints can also be affected (OMIM, 97). This can lead to serious damage of tissues, symptoms include abdominal pain, diabetes mellitus, enlargement of the heart leading to congestive heart failure and dysrhythmias, enlargement and cirrhosis of the liver which, if left untreated can over time lead to liver cancer (PGD, 1254). Hemochromatosis can also manifest in symptoms such as nausea, memory loss, and disorientation (Green, 97). It is frequently misdiagnosed as chronic hepatitis. This disorder is also highly treatable. The patient is put on a regular program of blood donation. Most patients experience total relief of symptoms using this treatment (PGD, 1254).<P>
Hemochromatosis is caused by the inheritance of two abnormal genes (autosomal recessive). The task of locating this gene has been challenging (Ponka, 97). It is known that the gene is on the short arm of chromosome 6 (Bacon, 96). The suspected region is very large. One of the genes in the area a suspect and was named HLA-H and is highly suspected to play a part in hemochromatosis. No definite proof has been presented (Ponka,97).<P>  
 
Bacon, Bruce (1996). The Elusive Hemochromatosis Gene and Other Reports on Iron Metabolism.   	http://sadeio.ucsf.edu/ALF/ALFfinal/proghemochrom.html<P>

Green, Hadley (1997). The Haemochromatosis Society. http://ourworld.compuserve/homepages/ghsoc/<P>

Online Mendelian Inheritance in Man (1997). Hemochromatosis, Hereditary:HH.  
               http://www3.ncbi.nlm.nih.gov:80/htbin-post/Omim/dispmim?235200<P>

Ponka, Prem (1997). Has the Hemochromatosis Gene been Identified? http://home.iSTAR.ca/chcts/ponmar97.
	html<P>

Professional Guide to Diseases, 3rd Edition. Springhouse Corportation. 1989.<P>


=========yorg=========
